### PR TITLE
chore: Add request source and ODFV to version 0.19 test repo

### PR DESCRIPTION
**Pull request by felixwang9817**
Signed-off-by: Felix Wang <wangfelix98@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by adchia** [comment]
probably worth deprecating the flags now too and just outputting that warning instead

**Comment by felixwang9817** [comment]
> probably worth deprecating the flags now too and just outputting that warning instead
yup can do that in a future PR

File: sdk/python/tests/example_repos/example_feature_repo_version_0_19.py [hunk 1]
```diff
@@ -50,3 +53,27 @@
     batch_source=global_daily_stats,  # Changed to `source` in 0.20
     tags={},
 )
+
+
+request_source = RequestSource(
```
[the remark this answers is not in the document]
Comment (felixwang9817) [reply sdk/python/tests/example_repos/example_feature_repo_version_0_19.py]: good catch, updated

File: sdk/python/tests/example_repos/example_feature_repo_version_0_19.py [hunk 2]
```diff
@@ -50,3 +54,27 @@
     batch_source=global_daily_stats,  # Changed to `source` in 0.20
     tags={},
 )
+
+
+request_source = RequestDataSource(
+    name="conv_rate_input", schema={"val_to_add": ValueType.INT64},
```
Comment (kevjumba) [inline sdk/python/tests/example_repos/example_feature_repo_version_0_19.py:60]: Request data source also uses lists now for schema
e.g 
```
schema=[
        Field(name="val_to_add", dtype=Int64),
        Field(name="val_to_add_2", dtype=Int64),
        Field(name="avg_daily_trips", dtype=Int64),
    ],
```

File: sdk/python/tests/example_repos/example_feature_repo_version_0_19.py [hunk 3]
```diff
@@ -50,3 +54,27 @@
...
+)
+
+
+@on_demand_feature_view(
+    inputs={
```
Comment (kevjumba) [inline sdk/python/tests/example_repos/example_feature_repo_version_0_19.py:65]: same here, inputs use lists 
`@on_demand_feature_view(
    sources=[
        driver_hourly_stats_view,
        input_request,
    ],
    schema=[
        Field(name="conv_rate_plus_val1", dtype=Float64),
        Field(name="conv_rate_plus_val2", dtype=Float64),
    ],
)`

**Review by adchia — APPROVED** [review]
/lgtm

**Pull request merged**